tests/crypto/cert_tests.c

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/enclave.h>
+#include <openenclave/internal/evidence.h>
+#include <openenclave/internal/report.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include <string.h>
+#if defined(OE_USE_OPENSSL)
+#include "../../../enclave/crypto/openssl/cert.h"
+#endif
+
+/* Test the internal API that is only avaiable on OpenSSL-based implementation.
+ */
+#if defined(OE_USE_OPENSSL)
+static void _test_X509_parse_name()
+{
+    X509_NAME* name = NULL;
+    char buf[256];
```

Comment:
Use constants of defines instead of magic numbers where possible for readability - this value seems to be based off `OE_X509_MAX_NAME_SIZE` for example, and may warrant its inclusion in an internal header like enclave/crypto/openssl/cert.h